test/test_roslaunch/test/params_basic.py

```diff
@@ -109,7 +109,11 @@ def test_commandandfile(self):
             text_data = f.read()
         with open(os.path.join(dir, 'resources', 'example.launch'), 'rb') as f:
             binary_data = f.read()
-        self.assertEquals(get_param("commandoutput"), binary_data)
+
+        # test 'command' attribute
+        if os.name != 'nt':  # skip testcase for `cat` command in Windows
+            self.assertEquals(get_param("commandoutput"), text_data)
```

Comment:
Shouldn't the last parameter be `binary_data`?

Reply:
Yes, you are right. It should be `binary_data`.